### PR TITLE
Quick Start second draft

### PR DESCRIPTION
**Pull request by rosemarymarano**
I think this draft is close to final form. Please review it for accuracy and completeness.

### CODE REVIEW

File: site/quickstart.md [hunk 1]
```diff
@@ -138,9 +138,13 @@ a.	Remove the domain's Ingress by using `helm`:
 ```
 helm delete --purge domain1-ingress
 ```
-b.	Remove the domain resource by using the sample [`delete-weblogic-domain-resources`](../kubernetes/samples/scripts/delete-domain/delete-weblogic-domain-resources.sh) script.
+b.	Remove the domain resources by using the sample [`delete-weblogic-domain-resources`](../kubernetes/samples/scripts/delete-domain/delete-weblogic-domain-resources.sh) script.
 
 c.	Use `kubectl` to confirm that the server pods and domain resource are gone.
+```
+kubectl get pods -n sample-domains-ns1
```
Comment (mriccell) [inline site/quickstart.md]: helm and kubectl commands should have a '$' in front.

File: site/quickstart.md [hunk 2]
```diff
@@ -138,9 +138,13 @@ a.	Remove the domain's Ingress by using `helm`:
 ```
 helm delete --purge domain1-ingress
```
Comment (mriccell) [inline site/quickstart.md]: helm and kubectl commands should have a '$' in front.

File: site/quickstart.md [hunk 3]
```diff
@@ -98,12 +98,12 @@ $ kubectl -n traefik get pod -w
...
 
-* Modifying the sample `inputs.yaml` file with the `domainUID` (`sample-domain1`) and domain namespace (`sample-domains-ns1`).
+* Modifying the sample `inputs.yaml` file with the `domainUID` (`sample-domain1`), domain namespace (`sample-domains-ns1`) and the base image (`oracle/weblogic:12213-patch-wls-for-k8s`).
 
-* Creating Kubernetes secrets `username` and `password` of the admin account using the [`create-weblogic-credentials`](../kubernetes/samples/scripts/create-weblogic-domain-credentials/create-weblogic-credentials.sh) script.
+* Creating Kubernetes secrets `username` and `password` using the [`create-weblogic-credentials`](../kubernetes/samples/scripts/create-weblogic-domain-credentials/create-weblogic-credentials.sh) script:
 ```
 cd ../kubernetes/samples/scripts/create-weblogic-domain-credentials
-./create-weblogic-credentials.sh -u <username> -p <password> -n sample-domain1-ns -d sample-domain1
```
Comment (mriccell) [inline site/quickstart.md]: commands should have a '$' in front to be consistent

File: site/quickstart.md [hunk 4]
```diff
@@ -98,12 +98,12 @@ $ kubectl -n traefik get pod -w
...
+./create-weblogic-credentials.sh -u <weblogic> -p <welcome1> -n sample-domain1-ns -d sample-domain1
 ```
 
 b.	Confirm that the operator started the servers for the domain:
```
Comment (mriccell) [inline site/quickstart.md]: when the user does kubectl get po -n sample-domain1-ns please specify that the user will see the admin server and managed servers running.

File: site/quickstart.md (hunk 1 above)
Comment (rosemarymarano) [reply site/quickstart.md]: Got it.

File: site/quickstart.md (hunk 3 above)
Comment (rosemarymarano) [reply site/quickstart.md]: Got it.

File: site/quickstart.md (hunk 4 above)
Comment (rosemarymarano) [reply site/quickstart.md]: Got it.

File: site/quickstart.md [hunk 5]
```diff
@@ -98,12 +98,12 @@ $ kubectl -n traefik get pod -w
...
 
-* Modifying the sample `inputs.yaml` file with the `domainUID` (`sample-domain1`) and domain namespace (`sample-domains-ns1`).
+* Modifying the sample `inputs.yaml` file with the `domainUID` (`sample-domain1`), domain namespace (`sample-domains-ns1`) and the base image (`oracle/weblogic:12213-patch-wls-for-k8s`).
 
-* Creating Kubernetes secrets `username` and `password` of the admin account using the [`create-weblogic-credentials`](../kubernetes/samples/scripts/create-weblogic-domain-credentials/create-weblogic-credentials.sh) script.
+* Creating Kubernetes secrets `username` and `password` using the [`create-weblogic-credentials`](../kubernetes/samples/scripts/create-weblogic-domain-credentials/create-weblogic-credentials.sh) script:
 ```
 cd ../kubernetes/samples/scripts/create-weblogic-domain-credentials
-./create-weblogic-credentials.sh -u <username> -p <password> -n sample-domain1-ns -d sample-domain1
+./create-weblogic-credentials.sh -u <weblogic> -p <welcome1> -n sample-domain1-ns -d sample-domain1
```
Comment (bhavaniravichandran) [inline site/quickstart.md]: Shouldn't the credentials be created before domain creation?

File: site/quickstart.md [hunk 6]
```diff
@@ -120,10 +120,10 @@ kubectl get pods -n sample-weblogic-operator1-ns
 ```
 
 c.	Create an Ingress for the domain, in the domain namespace, by using the [sample](../kubernetes/samples/charts/ingress-per-domain/README.md) Helm chart:
```
Comment (bhavaniravichandran) [inline site/quickstart.md]: Using the ingress-per-domain helm chart to create the ingress for the domain suffices, but I find using the host-routing.yaml in kubernetes/samples.charts/traefik/samples gives an idea of how to use the ingress with 2 domains.
kubectl create -f samples/host-routing.yaml
But the way it is right now works as well, I think.

File: site/quickstart.md [hunk 7]
```diff
@@ -98,12 +98,12 @@ $ kubectl -n traefik get pod -w
 
 a.	Create a new image with a domain home by running the [`create-domain`](../kubernetes/samples/scripts/create-weblogic-domain/domain-home-in-image/create-domain.sh) script. Follow the directions in the [README](../kubernetes/samples/scripts/create-weblogic-domain/domain-home-in-image/README.md) file, including:
```
[the remark this answers is not in the document]
Comment (rosemarymarano) [reply site/quickstart.md]: This quick start is a walk-through of a single flow (we tried to pick an easy one, domain-in-image). There will be extensive documentation on the other methods and more on this one as well. This doc is purely a quick and easy step-by-step for users to follow to get up and running with the operator.

**Pull request merged**